modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -20,6 +20,7 @@ const (
 	metricCallsTotal      = "traces_spanmetrics_calls_total"
 	metricDurationSeconds = "traces_spanmetrics_latency"
 	metricSizeTotal       = "traces_spanmetrics_size_total"
+	targetInfo            = "traces_spanmetrics_target_info"
```

Comment:
Should that be just `target_info`? 

OTel Upstream: span metrics connector + prometheus remote write will generate just `target_info`.